deployments/cli/community/install.sh

```
@@ -365,11 +365,48 @@ function startServices() {
     fi
 
     local api_container_id=$(docker container ls -q -f "name=$SERVICE_FOLDER-api")
+
+    # Verify container exists
+    if [ -z "$api_container_id" ]; then
+        echo "   Error: API container not found. Please check if services are running."
+        exit 1
+    fi
+
     local idx2=0
-    while ! docker exec $api_container_id python3 -c "import urllib.request; urllib.request.urlopen('http://localhost:8000/')" > /dev/null 2>&1;
-    do
-        local message=">> Waiting for API Service to Start"
-        local dots=$(printf '%*s' $idx2 | tr ' ' '.')    
+    local max_wait_time=300  # 5 minutes timeout
+    local start_time=$(date +%s)
+
+    echo "   Waiting for API Service to be ready..."
+    while ! docker exec "$api_container_id" python3 -c "
+        import urllib.request
```

The API wait loop has Python indentation issues, causing it not to exit properly even after the container starts.

The import statements in the multi-line Python string are incorrectly indented, causing silent failures.

Replacing it with a simple one-liner might fix the issue.

for e.g.
```
while true; do
        # Check if API is ready using single-line Python
        if docker exec "$api_container_id" python3 -c "import urllib.request; urllib.request.urlopen('http://localhost:8000/', timeout=5)" > /dev/null 2>&1; then
            # API is ready, break out of the loop
            break
        fi

```

---

@akshat5302 The indentations have been updated to one-liner.